app/models/facility.rb

```diff
@@ -209,4 +209,8 @@ def valid_block
       errors.add(:zone, "not present in the facility group")
     end
   end
+
+  def syncable_patients
+    registered_patients.with_discarded
+  end
```

Comment:
Is this a bit dissonant? "Syncable" in general means you are registered in, assigned to, or plan to visit. We're using a simplified definition here because it doesn't really matter when used for facility prioritization. However, does this set us up to do things like facility-level sync incorrectly?

Reply:
To address the dissonance, I suspect we'd either:
* update the query here to include all three clauses - This doesn't seem correct because it doesn't maintain feature parity in the sync API, and is also less performant probably
* update the name of this method? - Something along the lines of `prioritized_patients` or something? Making it clear that this boundary is not a facility-level sync boundary, but a looser boundary _within_ another block-or-FG-level sync?

Reply:
`prioritized_patients` works I think 👍🏼